validphys2/src/validphys/config.py

```diff
@@ -781,6 +782,8 @@ def produce_loaded_theory_covmat(
         """
         if theory_covmat_flag is False:
             return 0.
+        if len(glob.glob(output_path/"*theory_covmat*")) > 1:
+            raise ValueError("Too many theory_covmat files in folder tables")
```

Comment:
Sorry to be a pain in the neck but since you have a flag that allow you to use two different theorycovmat files I think it is perfectly fine to have more than one in the folder.

Not only that, I don't see why having a file called `I_really_hate_theory_covmat_oh_my_god` should be an impediment for me to run fits with a theory covmat.

All that to say that I would remove these two lines.